.github/workflows/release.yml

```diff
@@ -308,6 +308,21 @@ jobs:
         env:
           COMMITTER_TOKEN: ${{ secrets.HOMEBREW_GITHUB_API_TOKEN }}
 
+  winget_update:
+    name: Update Winget Manifest
+    runs-on: windows-latest
+    needs: [release_please, github_build, upload_artifacts]
+    if: ${{ needs.release_please.outputs.release_created == 'true' }}
+    steps:
+      - run: |
+          $releaseUrl = ${{ needs.release_please.outputs.html_url }}
+          $urlPrefix = $releaseUrl.Insert($releaseUrl.lastIndexOf('/'), '/download')
+          $url64 = Write-Host $urlPrefix/starship-x86_64-pc-windows-msvc.msi
+          $url32 = Write-Host $urlPrefix/starship-i686-pc-windows-msvc.msi
```

Comment:
Didn't know if it's preferable to take this fancy approach to building the MSI URLs or to simply hardcode them.

Reply:
I think hardcoding is simpler here. Is it possible to make the winget updater add the changelog?

Reply:
Changed to hardcoding: 7c33392

It looks like release notes for packages are a future enhancement available once they adopt their v1.1 manifest schema:
- https://github.com/microsoft/winget-cli/issues/1243
- https://github.com/microsoft/winget-create/issues/103